teradatalabs/cdh5-hive-kerberized/files/supervisord.d/hive-server2.conf

```diff
@@ -0,0 +1,8 @@
+[program:hive-server2]
```

Comment:
Why do we need custom command line parameters? Can't those be set in a config file?